commands/run.go

```diff
@@ -21,6 +22,11 @@ var RunCmd = cli.Command{
 			Name:   "terminator",
 			Hidden: true,
 		},
+		cli.StringFlag{
+			Name:  "encoding",
+			Usage: "Specify the output format, current options JSON and text",
+			Value: "text",
+		},
```

Comment:
We should make this a global flag under the app in main.go so that it works for other commands. The format error checking below can take place in the `Before` block as well.